flecsi/util/array_ref.hh

```diff
@@ -115,9 +115,9 @@ private:
 };
 
 template<class C>
-span(C &)->span<typename C::value_type>;
+span(C &) -> span<typename C::value_type>;
```

Comment:
This isn't how (our) `clang-format` lays this out (so our CI will just reject it).

Reply:
What version of clang-format do you use? I was using V11.

Reply:
It's 8.0.1, I think.  Obviously it would make sense to upgrade at some point, but that shouldn't be part of these changes.  It's also worth pointing out that I have a nifty script to reformat every commit on a branch; rather than having you figure out exactly which one to use I can just apply that before we merge this if you like.

Reply:
Thanks! I'll get back to you on this, if necessary.